static/components/App-around.js

```diff
@@ -21904,7 +21904,7 @@
 	    value: function render() {
 	      return _react2.default.createElement(
 	        'div',
-	        { className: 'card' },
+	        { id: 'card_' + this.props.details._id, className: 'card' },
```

Comment:
hum… 210907?!?
Is that a long num reference or something else?
I know it cannot be a line number… 🙏